README.md

```diff
@@ -2,40 +2,53 @@ Arsenik
 ================================================================================
 
 Configure your keyboard (even if it is not programmable) with a
-beginner-friendly, [Miryoku][1]-like approach to minimize finger movements!
-
-You can choose your options:
-- [angle-mod](https://colemakmods.github.io/ergonomic-mods/angle.html) (by
-default the only option activated)
-- 3 home-row mods (HRM) per hand for <kbd>Ctrl</kbd>, <kbd>Alt</kbd>, <kbd>Super</kbd>
-- 3 layer-tap keys under the thumbs: <kbd>Alt</kbd> (or <kbd>Shift</kbd> in 
-HRM)/<kbd>Backspace</kbd>, <kbd>Navigation</kbd>/<kbd>Space</kbd>,
-<kbd>Symbol</kbd>/<kbd>Return</kbd>
+beginner-friendly, simplified [Miryoku][1]-like approach to minimize finger
+movements!
 
 ![base, navigation and sym layers on a 33-key keyboard](img/all.svg)
 
+*Note: The keyboard layout presented here in the illustration is Qwerty but it
+works with others layouts as well — e.g. Azerty, Ergo‑L, Qwertz, Bépo, …*
+
 **Bring the keys to your fingers, rather than moving your fingers to the keys!**
 
 - A long press on the <kbd>Return</kbd> key brings up the <kbd>Symbol</kbd>
-layer, where all programming symbols are arranged for comfort and efficiency.
+layer in blue, where all programming symbols are arranged for comfort and
+efficiency.
 - A long press on the <kbd>Space</kbd> bar brings up the <kbd>Navigation</kbd>
-layer, with a numpad, cursor navigation (<kbd>ESDF</kbd>) and one-hand shortcuts.
+layer in orange, with a numpad, cursor navigation (<kbd>ESDF</kbd>) and one-hand
+shortcuts.
 
-This is how modern ergonomic keyboards work — e.g. [Planck][47], [Atreus][44],
-[Corne][42], [Ferris][34]… The goal here is to propose an approach that works
+This is how modern ergonomic keyboards work — e.g. [Planck][2], [Atreus][3],
+[Corne][4], [Ferris][5]… The goal here is to propose an approach that works
```

Comment:
FTR, the numbers that were used here referred to the number of keys for each keyboard model. And I kinda liked it that way. :-) 
However, if GFM supports refs like `[Planck][]` (and a link definition with `[Planck]: https://…`), that would be a nice readability improvement.